tests/mleap/test_mleap_model_export.py

```diff
@@ -218,8 +218,8 @@ def test_model_save_load_with_metadata(spark_model_iris, model_path):
         metadata={"metadata_key": "metadata_value"},
     )
 
-    reloaded_model = mlflow.pyfunc.load_model(model_uri=model_path)
-    assert reloaded_model.metadata.metadata["metadata_key"] == "metadata_value"
+    reloaded_model = Model.load(os.path.join(model_path, "MLmodel"))
```

Comment:
does this fail to load as a pyfunc?

Reply:
Yes, according to the first note in the [api docs](https://www.mlflow.org/docs/latest/python_api/mlflow.mleap.html), it's not possible to load the MLeap model flavor in Python. That explains why the previous test run failed.

Reply:
Yikes, sorry, been a long week. Yes, loading a JVM object in Python would be... problematic. Thanks for the fix